shenyu-kubernetes-controller/src/main/java/org/apache/shenyu/k8s/parser/DubboIngressParser.java

```diff
@@ -292,7 +292,7 @@ private List<DubboUpstream> parseUpstream(final V1IngressBackend backend, final
                         DubboUpstream upstream = DubboUpstream.builder()
                                 .upstreamUrl(clusterIP + ":" + defaultPort)
                                 .weight(100)
-                                .protocol("http://")
+                                .protocol("dubbo://")
```

Comment:
can the user config protocol?

Reply:
Users cannot set their own protocols for the time being.